bloom_nofos/nofos/templates/nofos/nofo_history.html

```diff
@@ -0,0 +1,106 @@
+{% extends 'base.html' %}
+{% load static nofo_name %}
+
+{% block title %}
+  Audit History for "{{ nofo|nofo_name }}"
+{% endblock %}
+
+{% block body_class %}nofo_history nofo_status--{{ nofo.status }}{% endblock %}
+
+{% block content %}
+  {% with nofo|nofo_name as nofo_name_str %}
+    {% with "Edit “"|add:nofo_name_str|add:"”" as back_text %}
+      {% url 'nofos:nofo_edit' nofo.id as back_href %}
+      {% include "includes/page_heading.html" with title="Audit history for “"|add:nofo_name_str|add:"”" back_text=back_text back_href=back_href only %}
+    {% endwith %}
+  {% endwith %}
+
+
+<div class="margin-bottom-5">
+  <span class="usa-tag usa-tag--big bg-group bg-group--not-uppercase bg-group--{{ nofo.group }}">{{ nofo.get_group_display }}</span>
+</div>
+
+<table class="usa-table" id="audit-events-table">
+  <caption>
+    <div>
+      <h2 class="margin-bottom-0">Recent Changes</h2>
+    </div>
+  </caption>
+  <thead>
+    <tr>
+      <th scope="col">Event</th>
+      <th scope="col">Object</th>
+      <th scope="col">User</th>
+      <th scope="col">Timestamp</th>
+    </tr>
+  </thead>
+  <tbody id="audit-events-body">
+    {% for event in audit_events %}
+    <tr>
+      <td>{{ event.event_type }}</td>
+      <td>{{ event.object_type }} - {{ event.object_description }}</td>
+      <td>{{ event.user }}</td>
+      <td>{{ event.timestamp }}</td>
+    </tr>
```

Comment:
Let's add the timezone thing here for EST, since that's what we do on the index page (also most/all of our users are in EST).

Reply:
good call, added that